app/control.c

```diff
@@ -709,6 +720,41 @@ eth_link(link_action_t action, int ifid, struct ether_addr *lladdr, int mtu,
 	return 0;
 }
 
+static int
+add_neighbor4(neighbor_struct_t *neighbor_struct, struct in_addr *ip,
```

Comment:
the function name should be more explicit about the fact that the neighbor is invalid